legal-api/migrations/versions/3d8eb786f4c2_add_coop_firm_sequence.py

```diff
@@ -17,8 +17,10 @@
 
 
 def upgrade():
-    op.execute("CREATE SEQUENCE legal_entity_identifier_coop START 1002395")
+    op.execute("CREATE SEQUENCE legal_entity_identifier_coop")
```

Comment:
I removed the starting sequence as the migration script(`transfer_to_new_lear.sql`) will update the sequence accordingly after transferring data from the old database into new empty database.